_posts/2017-09-05-Rust-2017-Survey-Results.md

```diff
@@ -0,0 +1,126 @@
+
+This year we had **5,368 responses**.  *That's more than 2,000 responses than we had last year*!
+
+The scale of the feedback was both inspiring and humbling, and we've worked hard to read through all of your comments and suggestions.  There were so many helpful ideas and experiences that people shared, and we truly do appreciate it.  Without further ado, let's take a look.
+
+![Chart: 66.9% Rust users, 9.9% stopped using, 23.2% never used](/images/2017-09-05-Rust-2017-Survey-Results/do_you_use_rust.png)
+
+Just as we saw last year, 2/3rds of responses are from Rust users and the remainder from non-users. This year, we separated out the "don't use Rust" to capture those who used Rust and stopped from those who never used Rust. It's inspiring to see so many developers wanting to help us make Rust better (even if they don't use it!) so that it can reach even more people.
+
+We'll talk more about Rust non-users later in this report, but first let's look at the responses from Rust users.
+
+# Using Rust
+
+![Chart: 0.5% less than a day, 4.2% less than a week, 13.1% less than a month, 39.7% less than a year, 42.5% over a year](/images/2017-09-05-Rust-2017-Survey-Results/how_long_using_rust.png "0.5% less than a day, 4.2% less than a week, 13.1% less than a month, 39.7% less than a year, 42.5% over a year")
+*(hover for more info)*
+
+This year, we're seeing a growing amount of experienced users sticking with Rust, with the "more than a year" users growing to over 42% (up from 30% from last year).  The beginners are also an impressively large set, with the "less than a month" crowd at just about 18%, meaning we're currently attracting nearly a 1/5th of our user base size, even as it grows larger, every month.
+
+![Chart: 36.5% less 1000 lines, 46.3% 1000 to 10000 lines, 14.2% 10000 to 100000 lines, 2.1% over 100000, 0.9% don't know](/images/2017-09-05-Rust-2017-Survey-Results/size_of_rust_projects.png "36.5% less 1000 lines, 46.3% 1000 to 10000 lines, 14.2% 10000 to 100000 lines, 2.1% over 100000, 0.9% don't know")
+*(hover for more info)*
+
+People are working ever-larger amounts of Rust, with medium- and large-scale lines of code totals both nearly doubling since last year as a percentage of the whole, now making up 16% of respondents (up from last year's 8.9%).  This shows a growing interest in using Rust in ever-larger projects, and a growing need for tools to support this growth.
+
+![Chart: 17.5% daily, 43.3% weekly, 24.4% monthly, 14.9% rarely](/images/2017-09-05-Rust-2017-Survey-Results/how_often_use_rust.png "17.5% daily, 43.3% weekly, 24.4% monthly, 14.9% rarely")
+
+Despite the rising amount of code developers are working with, we're seeing a small downtick in both daily and weekly users.  Daily users have fallen from 19% to 17.5%, and weekly users have fallen from 48.8% to 43.3%. This could be a natural transition in this stage of our growth, as a broader range of developers begin using Rust.
```

Comment:
I would make clear that the absolute number of daily users is growing, even though the percentage is falling